src/components/Profile View/ProfileColor.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react"
+import AxiosAuth from "../Auth/axiosWithAuth"
+import { TwitterPicker } from "react-color"
+
+const ProfileColor = () => {
+  const [color, setColor] = useState({
+    color: "",
+  })
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    AxiosAuth()
+      .put(`https://shopping-cart-be.herokuapp.com/api/store/`, color)
+      .then((res) => console.log("ProfileColorSubmit", res.data))
```

Comment:
Is there a reason for there only being a console.log in your .then?

Reply:
Yeah to see if it is successfully changed.